docs/source/reproducers/01-considerations.md

```diff
@@ -0,0 +1,64 @@
+# Environment reproducers general considerations
+## Tested environment
+This reproducer has been successfully tested on a CentOS Stream 9 hypervisor
+running latest CRC qcow2, and latest CentOS Stream 9 qcow2 images.
```

Comment:
Is the testing continuous? If not the 'latest' may change.

Reply:
We'll eventually get a periodic (daily, or weekly) job testing the reproducer capabilities. It's of course not easy, but that's something we want to provide, in order to prove the reproducer is working as it should.